sem1/1.py

```diff
@@ -1 +1,9 @@
-#  Найдите сумму цифр трехзначного числа. 
\ No newline at end of file
+#  Найдите сумму цифр трехзначного числа. 
+m=int(input('Введите трехзначное число '))
+if m<1000 and m>99:
+    m1=m//100
+    m2=m%100//10
+    m3=m%10
+    print(m1+m2+m3)
```

Comment:
Полное отсутствие пробелов. Пожалуйста, переведите код к виду PEP8.
```suggestion
m = int(input('Введите трехзначное число '))
if m<1000 and m > 99:
    m1 = m // 100
    m2 = m % 100 // 10
    m3 = m % 10
    print(m1 + m2 + m3)
```

Reply:
Решение верное